delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resolvers/Resolver.scala

```diff
@@ -49,6 +64,28 @@ object Resolver {
       projects: NonEmptyList[ProjectRef],
       identityResolution: IdentityResolution,
       priority: Priority,
+      source: Json,
       tags: Map[Label, Long]
   ) extends Resolver
+
+  val context: ContextValue = ContextValue(contexts.resolvers)
+
+  implicit val resolverEncoder: Encoder.AsObject[Resolver] = {
+    Encoder.AsObject.instance {
+      case InProjectResolver(_, _, priority, _, _)                                                 =>
+        JsonObject(
+          "priority" -> priority.asJson
+        )
+      case CrossProjectResolver(_, _, resourceTypes, projects, identityResolution, priority, _, _) =>
+        JsonObject(
+          "priority"      -> priority.asJson,
+          "resourceTypes" -> resourceTypes.asJson,
```

Comment:
If you want this field to be compacted, you can do the following:
```scala
// Declare this somewhere on the top or bottom of the companion object
private val ctx = JsonLdContext(ContextValue.empty, base = Some(nxv.base))

// then
"resourceTypes" -> resourceTypes.map(ctx.compact(_, useVocab = false)).asJson

```

Reply:
The resolver has more fields including the source and tags so they are not the same thing and it is not possible to generate a Resolver only from a ResolverValue and an Iri

Reply:
For that purpose it can be generated with empty tags and source, since for the encoding they are discarded. 

Alternatively you can refactor some of this stuff. But basically this `Encoder.AsObject[Resolver]` generates the same Json as the other one